src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -25,7 +25,7 @@
     <ItemGroup>
      <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)CSharp.ProjectItemsSchema.xaml;"/>
 
-     <ProjectCapability Include="CSharp;Managed"/>
+     <ProjectCapability Include="CSharp;Managed;ClassDesigner;"/>
```

Comment:
Don't put a trailing ";"

Reply:
We put a trailing ; in our other targets. I was making this consistent.